package.json

```diff
@@ -100,7 +101,10 @@
     "@changesets/cli": "^2.13.1",
     "@commitlint/cli": "16.2.1",
     "@commitlint/config-conventional": "16.2.1",
-    "@emotion/babel-preset-css-prop": "^10.0.14",
+    "@emotion/babel-preset-css-prop": "^11.2.0",
+    "@emotion/css": "^11.9.0",
+    "@emotion/eslint-plugin": "^11.7.0",
+    "@emotion/jest": "^11.9.1",
```

Comment:
`@emotion/jest@^11.9.1` summary of changes:
- `toHaveStyleRule` added a "prettify" step with adds whitespaces back to style rules. Checkout [an example of the resulting changes here.](https://github.com/twilio-labs/paste/pull/2376/files#diff-7bc2aa7fd1ba2a1e5bfb3264e84a55254ad1d87dabc745e15ba31ad942f1d594R68)
- Matcher compares the expected style rule to the serialized styles associated with an emotion class name. This broke tests for the `BaseRadioCheckbox` because the style rules did not include the sibling selectors with the class names. Check out the [relevant test spec file](https://github.com/twilio-labs/paste/pull/2376/files#diff-c1e03d28912ba7c46f00f8bde3ac408101f8cdec64231afdf6f104e85e480539R91) to see temp resolution.